### DISCUSSION TITLE
OPA helm chart

### DISCUSSION THREAD

**Discussion by bvamshidhar** [open-policy-agent/community#598] [Community]
I couldn't find helm chart for OPA installation for opa:0.64.1. Why doesn't OPA have a helm chart?

**Comment by anderseknert** [comment]
There's one provided as part of the [kube-mgmt](https://github.com/open-policy-agent/kube-mgmt) project. Perhaps you'll find it useful.

**Reply by bvamshidhar** [reply]
Hey. Yeah, I have seen this. I didn't want the kube-mgmt files. I was looking for a helm chart just for [OPA-Istio](https://www.openpolicyagent.org/docs/latest/envoy-tutorial-istio/)
I tried to create a custom chart with the [quick_start.yaml](https://raw.githubusercontent.com/open-policy-agent/opa-envoy-plugin/main/examples/istio/quick_start.yaml) manifest but had trouble with the MutatingWebhookConfiguration

**Reply by anderseknert** [reply]
Not sure about OPA/Istio, and I don't use Helm myself, but AFAIK you can use those charts for OPA only, and there's no requirement to add anything kube-mgmt related. Perhaps @eshepelyuk is able to fill in some of my blanks, as he's done a ton of work on those charts. And if there's anything missing from the charts, contributions are always welcome.
